code/datums/datacore.dm

```diff
@@ -117,14 +117,21 @@ GLOBAL_LIST_EMPTY(PDA_Manifest)
 				break
 
 	var/list/all_jobs = get_job_datums()
+	var/is_custom_job = TRUE
 
 	for(var/datum/job/J in all_jobs)
 		var/list/alttitles = get_alternate_titles(J.title)
 		if(!J)	continue
```

Comment:
Please do remove this, it does nothing

Reply:
if an element is not present in a for loop the loop does not run, and it would runtime on the line above anyway even if it did

Reply:
It also should be in the style of
```
if(!J)
	continue
```
In accordance to our code standards.
But yes, this check will do nothing as S34N said.

Reply:
ok